remote/src/main/java/com/redvelvet/remote/service/MovieApiService.kt

```diff
@@ -1,5 +1,56 @@
 package com.redvelvet.remote.service
```

Comment:
should we make the requests of making seession and token into its service seperated from movies api?

Reply:
because it is related to the user authintication not to the movies

Reply:
```suggestion
package com.redvelvet.remote.service
interface AuthenticationService {
    @GET("authentication/token/new")
    suspend fun getRequestToken(): Response<TokenDto>

    @POST("authentication/token/validate_with_login")
    suspend fun validateRequestTokenWithLogin(@Body loginRequest: LoginRequest): Response<TokenDto>

    @FormUrlEncoded
    @POST("authentication/session/new")
    suspend fun createSession(@Field("request_token") requestToken: String): Response<SessionDto>

    @GET("authentication/guest_session/new")
    suspend fun createGuestSession(): Response<GuestSessionDto>

    @FormUrlEncoded
    @HTTP(method = "DELETE", path = "authentication/session", hasBody = true)
    suspend fun deleteSession(@Field("session_id") sessionId: String): Response<BaseResponse>
}

```

Reply:
Done